recipes/galaxy-upload/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "1.0.0" %}
+{% set sha256 = "d307bdf1c549e8aa41480b94d8aff9828df4019f96fe5fa4290eb696d422f87c" %}
+
+package:
+  name: galaxy-upload
+  version: {{ version }}
+
+source:
+  url: https://github.com/galaxyproject/galaxy-upload/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv"
```

Comment:
Please add [entry points](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points) for `galaxy-upload` and `galaxy-history-search`.